scripts/variantstore/wdl/extract/vds_validation.py

```diff
@@ -1,6 +1,6 @@
 import argparse
 import hail as hl
-
+import itertools
```

Comment:
TODO! This is a straight copy from EchoCallset. To be reviewed.

Reply:
Yes--you'll note that there are several steps of validation that we do that have been commented out. This is because the Echo VDS passed all of those "tests" and then was falling over in the last test of the validation. So all but the last test are commented out here (which I dont think we want to maintain) and then the final test is broken down by chromosome, which I do think we might as well keep since for now, VDS is scale driven, and the next callset is likely not going to be smaller